src/components/pages/Home/UseCasesSection/index.tsx

```diff
@@ -297,24 +358,55 @@ const UseCasesSection: React.ForwardRefRenderFunction<HTMLElement> = () => (
                       </Tooltip>
                     </Code>
                     <ExampleBox title="CML Report">
-                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative/cml_tensorboard_case/pull/1">
+                      <a target="_blank" rel="noreferrer" href="https://github.com/iterative-test/cml-example-tensorboard/pull/1">
                         <Image src="/img/github/tensorboard-report.png" alt="GitHub Tensorboard report example" />
                       </a>
                     </ExampleBox>
-
+        
+                  </Collapser>
+                )}
+                bitbucket={(
+                  <Collapser>
+                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-test/cml-example-tensorboard">
```

Comment:
- [ ] TODO: @0x2b3bfa0 

```suggestion
                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-ai/cml-example-tensorboard">
```

Reply:
```suggestion
                    <Code filename="bitbucket-pipelines.yml" repo="https://bitbucket.org/iterative-ai/cml-tensorboard-case">
```